src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue

```diff
@@ -39,9 +40,20 @@
 			</slot>
 		</span>
 
-		<span v-if="$slots.default" class="checkbox-content__text" :class="[textClass]">
-			<!-- @slot The checkbox/radio label -->
-			<slot />
+		<span class="checkbox-content__wrapper">
+			<span
+				v-if="$slots.default"
+				:id="labelId"
+				class="checkbox-content__text"
+				:class="textClass">
+				<!-- @slot The checkbox/radio label -->
+				<slot />
+			</span>
+			<em v-if="!isButtonType && ($slots.description || description)" :id="descriptionId" class="checkbox-content__description">
```

Comment:
I am not sure `em` is a proper tag here as semantically it `represents stress emphasis of its contents` so I do not see how this should be semantically more emphasized than the label. would just use a `<span>` as well here.